.github/chronographer.yml

```diff
@@ -1,30 +1,10 @@
----
-
+branch-protection-check-name: Changelog entry
 action-hints:
-  # check-title-prefix: chng  # default: `{{ branch-protection-check-name }}: `
-  external-docs-url: https://pip.pypa.io/how-to-changelog
+  check-title-prefix: "Chronographer: "
+  external-docs-url: https://pip.pypa.io/dev/news-entry-failure
   inline-markdown: >
-    Check out https://pip.pypa.io/how-to-changelog
-
-branch-protection-check-name: Timeline protection
-
+    See https://pip.pypa.io/dev/news-entry-failure for details.
 enforce-name:
-  # suffix: .md
   suffix: .rst
-
-exclude:
-  bots:
-  - dependabot-preview
-  - dependabot
-  - patchback
-  humans:
-  - pyup-bot
```

Comment:
@pradyunsg FYI the "humans" list makes the bot treat said _regular_ GitHub accounts as if they were automations. This is useful for things that are not implemented as GitHub Apps but rather use PAT (personal auth token) of a normal GitHub account. The bot has no way of distinguishing such automations automatically, based on the event/API data which is why such a list is needed.

_Remember this for the future, should such an automation be added._